package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "palette",
   "private": true,
+  "engines": {
+    "node": "10.x"
+  },
```

Comment:
Is this necessary? We shouldn't have to lock our node version for palette.

Reply:
I think engines is probably the wrong place for this. An nvmrc is better. But you can't yarn install here in 12 because palette-docs can't build. It actually prevents `yarn add` because it's trying to rebuild on add. (Due to the older sharp in the gatsby image plugins).

I'll switch this to nvmrc so it reflects the local development environment. But we should log some kind of ticket for cleaning up docs to make things installable.